src/window.rs

```diff
@@ -156,6 +156,21 @@ pub struct WindowAttributes {
   /// The default is `true`.
   pub resizable: bool,
 
+  /// Whether the window is minimizable or not.
+  ///
+  /// The default is `true`.
+  pub minimizable: bool,
+
+  /// Whether the window is maximizable or not.
+  ///
+  /// The default is `true`.
+  pub maximizable: bool,
+
+  /// Whether the window is closable or not.
+  ///
+  /// The default is `true`.
+  pub closable: bool,
```

Comment:
I didn't add these because none of the others have them – eg `set_resizable` has some quite extensive platform specific notes which aren't linked from here. But probably better to link anyway I guess 👍